docs/tracking-algorithms/track.rst

```diff
@@ -26,12 +27,127 @@ For full details of the TRACK API in TCTrack see the
 Installation
 ------------
 
-.. TODO
+Using the TRACK module in TCTrack requires TRACK to be installed on a user's system.
+
+TRACK requires NetCDF as well as a C compiler, a FORTRAN 77 compiler, Make, and
+`makedepend`. It can then be installed with the following commands:
+
+.. code-block:: bash
+
+  # Download the TRACK code
+  git clone https://gitlab.act.reading.ac.uk/track/track
+  cd track
+  git checkout 6ded301a5f5183d73e5b49c16019024b9a53eff7
```

Comment:
Add a note to say that we are pinning the commit to what has been tested for TCTrack, but newer might be available?